docs/README.md

```diff
@@ -63,10 +63,9 @@ an issue:
   * [Testing on Headless CI Systems (Travis, Jenkins)](tutorial/testing-on-headless-ci.md)
   * [DevTools Extension](tutorial/devtools-extension.md)
   * [Automated Testing with a Custom Driver](tutorial/automated-testing-with-a-custom-driver.md)
-* Packaging
-  * [Code Signing](tutorial/code-signing.md)
 * [Distribution](tutorial/application-distribution.md)
-  * [Support](tutorial/support.md)
+  * [Supported Platforms](tutorial/support.md#supported-platforms)
```

Comment:
Hm, this document contains more info than just the supported platforms and versions, it also contains info on how to get actual support. 🤔

Reply:
Ah ever mind, it's listed twice in the TOC.

Reply:
Yeah... alternatively, we could just remove this link.

Reply:
I'm OK with having it in there twice if the url links have different query sections. It wouldn't surprise me if people on this page were looking for info on supported platforms.